pydicom/dataset.py

```diff
@@ -759,24 +764,51 @@ def convert_pixel_data(self):
         if already_have:
             return
 
-        # Find all pixel data handlers that support the transfer syntax
-        suitable_handlers = [hh for hh in pydicom.config.image_handlers
-                             if hh and hh.supports_transfer_syntax(self)]
+        # Find all possible handlers that support the transfer syntax
+        transfer_syntax = self.file_meta.TransferSyntaxUID
+        possible_handlers = [hh for hh in pydicom.config.pixel_data_handlers
+                             if hh.supports_transfer_syntax(transfer_syntax)]
 
-        # No suitable handlers are available
-        if not suitable_handlers:
+        # No handlers support the transfer syntax
+        if not possible_handlers:
             raise NotImplementedError(
                 "Unable to decode pixel data with a transfer syntax UID of "
-                "'{0}' ({1}) as there are no suitable pixel data handlers "
-                "available. Please see the list of supported Transfer "
-                "Syntaxes in the pydicom documentation for information on "
-                "which additional packages might be required"
+                "'{0}' ({1}) as there are no pixel data handlers "
+                "available that support it"
                 .format(self.file_meta.TransferSyntaxUID,
                         self.file_meta.TransferSyntaxUID.name)
             )
 
```

Comment:
I think a "pointer" to more information is still needed here.

Reply:
That message only appears if there are no handlers at all that support the pixel data though, so directing them to the documentation won't help. I originally had a message saying they should start an issue on github but figured we'd probably get inundated by duplicate issues.

Reply:
Hmm, I see your point, but it might help if they have the opportunity to re-export the images with another transfer syntax from the original source.

Reply:
Done